test/distributed/tensor/experimental/test_tp_transform.py

```diff
@@ -74,13 +75,29 @@ def test_tp_transform_with_uncovered_op(self):
             exported_program = torch.export.export(
                 model, inputs, strict=True
             ).run_decompositions()
-        tp_exported_program = tensor_parallel_transformation(
-            exported_program,
-            self.rank,
-            self.world_size,
-            self.device_type,
-            {"fc": ColwiseParallel},
-        )
+
+        @contextlib.contextmanager
+        def temp_deregister():
+            import torch.utils._pytree as pytree
+            from torch.distributed.tensor._dtensor_spec import DTensorSpec
+
+            try:
+                pytree._deregister_pytree_node(DTensorSpec)
+                yield
+            finally:
+                pytree.register_constant(DTensorSpec)
+
+        # TODO: Having DTensorSpec in pytree causes issue with tensor_parallel_transformation
+        # Need to understand the interaction here
```

Comment:
@tugsbayasgalan 

Some thing weird is going on here. 

Also, I don't need `torch.utils._pytree.register_constant(DTensorSpec)` to make _dynamo_graph_capture_for_export passing. 

It's just needed for strict export.

Reply:
alright, I reverted `pytree.register_constant(DTensorSpec)`.

Reply:
Ohh interesting ok